__tests__/api.test.js

```diff
@@ -42,43 +64,53 @@ describe('GET requests', () => {
             expect(articleZero.article_id).toBe(2)
         })
     })
-    test.only('400 : handles a bad request with out of bounds :article_id', () => {
+    test('404 : handles an invalid request with out of bounds :article_id', () => {
         return request(app)
         .get('/api/articles/999')
-        .expect(400)
+        .expect(404)
         .then(({body : {msg}}) => { 
             expect(msg).toBe("Not Found")
         })
     })
+    test('400 : handles a bad request with an article_id that is NaN', () => {
+        return request (app)
+        .get('/api/articles/cheese')
+        .expect(500)
```

Comment:
This should be 400 - bad request. 500 is something that we have broken, not the client.